src/System.Threading.Thread/tests/MTAMain/MTAMain.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+namespace MTAMain
+{
+    internal static class Program
+    {
+        [MTAThread]
+        static int Main(string[] args)
+        {
+            int retValue = 0;
+
+            bool mode = bool.Parse(args[0]);
+            if (mode)
+            {
+                if (Thread.CurrentThread.GetApartmentState().Equals(ApartmentState.MTA))
```

Comment:
You can just do `...GetApartmentState() == ApartmentState.MTA)`. Equals is just a more verbose and more expensive way to achieve the same.